Modules/IO/Meta/test/itkMetaImageIOTest.cxx

```diff
@@ -90,6 +90,9 @@ itkMetaImageIOTest(int argc, char * argv[])
   myImage::Pointer image = reader->GetOutput();
   image->Print(std::cout);
 
+  // MetaImage is expected to have a modality tag
+  image->GetMetaDataDictionary().Get("Modality");
```

Comment:
Suppose that accidentally the dictionary does _not_ have a modality tag. What would `image->GetMetaDataDictionary().Get("Modality")` do? It seems to me that it would then silently return null, right?

Would it be possible to add an "assert" or "expect" statement here, to specify what is expected? Something like:

    ITK_TEST_EXPECT_TRUE(image->GetMetaDataDictionary().Get("Modality") != nullptr);

Or just:

    ITK_TEST_EXPECT_TRUE( image->GetMetaDataDictionary().HasKey("Modality") );

Reply:
@zivy What did you think of my suggestion to improve the test that you added?